### DISCUSSION TITLE
Where do you put the files at?

### DISCUSSION THREAD

**Discussion by SunlessTerror28** [dialogic-godot/dialogic#985] [Q&A]
Where do you put the files at in the app or the project file?

**Comment by coppolaemilio** [comment]
Not sure what you mean exactly, but if you are asking how to install:

To install a Dialogic, download it as a ZIP archive. All releases are listed here: [releases](https://github.com/coppolaemilio/dialogic/releases). Then extract the ZIP archive and move the `addons/` folder it contains into your project folder. Then, enable the plugin in project settings.

If you want to know more about installing plugins you can read the [Godot docs page](https://docs.godotengine.org/en/stable/tutorials/plugins/editor/installing_plugins.html).

You can also install Dialogic using the **AssetLib** tab in the editor.

Hope that answers your question!
